src/o365/spfx/commands/project/project-rename.ts

```diff
@@ -0,0 +1,193 @@
+
+    const replaceFileContent = (filePath: string) => {
+      if (fs.existsSync(filePath)) {
+        let existingContent = fs.readFileSync(filePath, 'utf-8');
+        let updatedContent;
+        if (filePath.endsWith('.json')) {
+          updatedContent = JSON.parse(existingContent);
+
+          if (updatedContent &&
+            updatedContent.name) {
+            updatedContent.name = args.options.newName;
+          }
+          if (updatedContent &&
+            updatedContent['@microsoft/generator-sharepoint'] &&
+            updatedContent['@microsoft/generator-sharepoint'].libraryName) {
+            updatedContent['@microsoft/generator-sharepoint'].libraryName = args.options.newName;
+          }
+          if (updatedContent &&
+            updatedContent['@microsoft/generator-sharepoint'] &&
+            updatedContent['@microsoft/generator-sharepoint'].solutionName) {
+            updatedContent['@microsoft/generator-sharepoint'].solutionName = args.options.newName;
+          }
+          if (updatedContent &&
+            updatedContent['@microsoft/generator-sharepoint'] &&
+            updatedContent['@microsoft/generator-sharepoint'].libraryId &&
+            args.options.generateNewId) {
+            updatedContent['@microsoft/generator-sharepoint'].libraryId = newId;
+          }
+          if (updatedContent &&
+            updatedContent.solution &&
+            updatedContent.solution.name) {
+            updatedContent.solution.name = updatedContent.solution.name.replace(new RegExp(projectName, 'g'), args.options.newName);
+          }
+          if (updatedContent &&
+            updatedContent.solution &&
+            updatedContent.solution.id &&
+            args.options.generateNewId) {
+            updatedContent.solution.id = newId;
+          }
+          if (updatedContent &&
+            updatedContent.paths &&
+            updatedContent.paths.zippedPackage) {
+            updatedContent.paths.zippedPackage = updatedContent.paths.zippedPackage.replace(new RegExp(projectName, 'g'), args.options.newName);
+          }
+          if (updatedContent &&
+            updatedContent.container) {
+            updatedContent.container = args.options.newName;
+          }
+          fs.writeFileSync(filePath, JSON.stringify(updatedContent, null, 2), 'utf-8');
+        } else {
+          updatedContent = existingContent.replace(new RegExp(projectName, 'g'), args.options.newName);
+          fs.writeFileSync(filePath, updatedContent, 'utf-8');
+        }
+        if (this.debug) {
+          cmd.log(`Updated ${filePath.split('/').pop()}`);
+        }
+      }
+    }
+
+    if (this.debug) {
+      cmd.log(`Renaming SharePoint Framework project to '${args.options.newName}'`);
+    }
+
+    filePaths.forEach((filePath) => {
+      replaceFileContent(filePath);
+    });
+
+    if (this.verbose) {
+      cmd.log('DONE');
+    }
+
+    cb(`SharePoint Framework project successfully renamed to '${args.options.newName}'`);
```

Comment:
When the command succeeds, we need to call `cb()` without any arguments. When you pass an argument, it's treated as an error message.

Reply:
Thanks. Removed the argument.